# this is a message that holds the force at the end effector

std_msgs/Header header
float64 force_1
float64 force_2
float64 force_3
float64 force_4
float64 force_5
float64 force_6
